docs/src/details/xml/tools/cli.rst

```diff
@@ -1,23 +1,194 @@
-
 .. _editor_cli:
 
 ===
 CLI
 ===
 
+The LVGL Editor provides a Node-based command line tool, ``lved-cli.js``, for generating/compiling code from XML, validating projects, syncing from Figma, comparing trees, and running headless UI tests.
+
+.. note::
+   Pre-release build. Intended for development and CI. Expires end of 2025.
+
+.. toctree::
+   :maxdepth: 2
+   :local:
+
+
 Overview
 ********
 
+- **Binary:** ``lved-cli.js`` (Node script)
+- **Platform:** Node 18+ recommended
+- **Container engine:** Podman
+- **Resources:** unpacked from ``lvgl-resources.zip`` to an internal directory
+
+
+Quick start
+***********
+
+Generate:
+
+.. code-block:: bash
+
+   lved-cli.js generate path/to/project
+
+Compile (web or node):
+
+.. code-block:: bash
+
+   lved-cli.js compile path/to/project --target web --start-service
+
+Validate (limit shown errors):
+
+.. code-block:: bash
+
+   lved-cli.js validate path/to/project --errorlimit 25
+
+Tests:
+
+.. code-block:: bash
+
+   lved-cli.js run-all-tests path/to/project
+   lved-cli.js run-test path/to/project tests/test-file.xml
+
+Figma sync:
+
+.. code-block:: bash
+
+   lved-cli.js figma-sync path/to/project --start-service
+
+
 Usage
 *****
 
-Arguments
----------
+.. code-block:: text
```

Comment:
It seems Usage and Commands has a lot of overlapping things. Please unify the two sections.